### DISCUSSION TITLE
Unable to resolve bookdown conflict

### DISCUSSION THREAD

**Discussion by Shruti-Kaushal** [jtr13/cc21fall1#82] [Q&A] [answered]
Hi,
I updated the bookdown file in the repo of #46 but I am unable to resolve the conflict. For some reason the conflict shows up but nothing is accessible (can't click on "resolve conflict" button)
How do I resolve the conflict?

Thank you!
Shruti

**Comment by jtr13** [comment, marked as answer]
This has been resolved... see https://github.com/jtr13/cc21fall1/pull/46
